Library/Homebrew/cmd/bottle.rb

```diff
@@ -183,6 +186,15 @@ def bottle_formula(f)
     relocatable = false
     skip_relocation = false
 
+    formula_source_time = f.stable.stage do
+      newest_mtime = nil
+      Pathname(Dir.pwd).find do |path|
+        newest_mtime ||= path.mtime
+        newest_mtime = path.mtime if path.mtime > newest_mtime
+      end
+      newest_mtime
+    end
```

Comment:
I was going to suggest

```
formula_source_time = Pathname(Dir.pwd).find.map(&:mtime).max
```
but then I realized that we are still stuck with Ruby 1.8.7. :sob: Could the (more compatible, more ugly)

```
formula_source_time = Pathname(Dir.pwd).to_enum(:find).map(&:mtime).max
```

still be a good enough and succinct replacement for this entire block?